packages/cms/src/utils/varSwapRecursive.js

```diff
@@ -46,7 +49,9 @@ const varSwapRecursive = (sourceObj, formatterFunctions, variables, query = {},
             obj[skey] = code;
           }
           catch (e) {
-            console.log("Error in ES5 transpiling in varSwapRecursive");
+            if (verbose) {
+              console.error(`Error in ES5 transpiling in varSwapRecursive.\nID was: ${obj.id}\nLogic attempted was: ${obj[skey]}`);
```

Comment:
- don't put this behind `verbose` (we will probably always want to show errors on prod servers)
- don't print the full logic (it's unnecessary if we have the ID and bloats stdout)
- make the message all on one line: `Error in ES5 transpiling in varSwapRecursive (ID ${obj.id})`
- is there any benefit in printing out `e.message` here?

Reply:
The reason I was printing the full logic was: 

- In case we use this transpiler in the future, perhaps for generic logic that isn't a visualization. Visualizations can be assumed to have an `id` but maybe some future case wouldn't.
- The id of a visualization is not present in the CMS at all. Seeing "error in id 3" would require the extra step of opening up your db directly to figure out which one I'm talking about (note that visualizations don't have plaintext names either).  Viewing the logic itself in the stacktrace was a way for me to give the user the "oh it's that one" helping hand.

Thoughts on that?

Regarding your second question, printing out `e.message` is indeed probably helpful here, because it would give more info on why the __transpiling__ failed, which is a different error from the OTHER error you will get later, which is why the es6 may have been malformed (handled in d3pluspropify)

Reply:
I understand, but I think that this is an opportunity to pass additional information to the function instead of printing out the logic (I'm worried about printing out like, ~50 line viz/generator logic here instead of just being able to say "hay it's viz #23"). I actually started passing through the viz ID to propify in datausa for this very reason: https://github.com/DataUSA/datausa-site/blob/master/app/components/Viz/index.jsx#L21

thoughts? still open to something other than that

Reply:
This sounds good to me.